virtool/references/oas.py

```diff
@@ -0,0 +1,917 @@
+from typing import Optional, Union, List
```

Comment:
Excellent use of validators in this module. But... we do need to test them. Add a `test_oas.py` to make sure that model implementing the validators work as expected.